crates/egui_demo_app/Cargo.toml

```
@@ -16,9 +16,10 @@ crate-type = ["cdylib", "rlib"]
 
 
 [features]
-default = ["glow", "persistence"]
+default = ["glow", "persistence", "image_viewer"]
```

This adds a lot of extra dependencies to users just doing `cargo run`

```suggestion
default = ["glow", "persistence"]
```